src/App.js

```diff
@@ -1,11 +1,83 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please **enable all** linter checks on your code, to ensure that your code is robust code and clean.